tests/jit_spec.rs

```diff
@@ -34,7 +34,7 @@ mod tests {
     #[tokio::test]
     async fn test_executor_nested() {
         //  NOTE: This test makes a real HTTP call
-        let request = Request::new("query {posts {title userId user {id name email} }}");
+        let request = Request::new("query {posts {title userId user {id name websiteExpr} }}");
```

Comment:
Add a separate test for nested IOs. Let's use [albums](https://jsonplaceholder.typicode.com/user/1/albums) as its a real working URL.

Reply:
IGNORE THIS.